.github/workflows/release.yml

```diff
@@ -75,6 +75,33 @@ jobs:
         bin/docker-retag-all $TAG main
         bin/docker-push main
   # todo: Keep in sync with `kind_integration.yml`
+  windows_static_cli_tests:
+    name: Static CLI tests (windows)
+    runs-on: windows-latest
+    needs: [docker_build]
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Try to load cached Go modules
+      # actions/cache@v1.1.2
+      uses: actions/cache@70655ec
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Download image archives
```

Comment:
@alpeb @kleimkuhler Just realised we dont actually push artifacts to Github during release right? which means that this would fail?

Reply:
Shoot, that's right :-(

Reply:
Yes that's correct we don't push artifacts in the release workflow so this would be an issue..

I think this could do something similar to `cloud_integration` where it installs from the locally built images: https://github.com/linkerd/linkerd2/blob/4372ed56dd39f79ba4fa8286e3f95d4383786f5d/.github/workflows/cloud_integration.yml#L89-L90?